build.rs

```diff
@@ -752,6 +788,22 @@ fn perlasm(
     }
 }
 
+fn make_prefix_headers(dst: &PathBuf) {
+    let symbol_prefix_includes = dst.join(SYMBOL_PREFIX_INCLUDE);
+    std::fs::create_dir(&symbol_prefix_includes).unwrap();
+    let args: Vec<String> = vec![
+        "run".into(),
+        "util/make_prefix_headers.go".into(),
+        "-out".into(),
+        symbol_prefix_includes
+            .to_str()
+            .expect("Could not convert path")
+            .into(),
+        "symbols.txt".into(),
+    ];
+    run_command_with_args(&get_command("GO_EXECUTABLE", "go"), &args);
```

Comment:
Does this mean that go is required now to build ring? Or has this already been the case? I'd be cautious with introducing dependencies on entire new programming languages.

Reply:
O never mind it is only being run during packaging: 25add85a541945b551a2b7ea7b4eaede71dbf507

Reply:
> Does this mean that go is required now to build ring? Or has this already been the case? I'd be cautious with introducing dependencies on entire new programming languages.

I had the same question. We need to make certain that Go isn't used as part of the build, only part of packaging/pregeneration.

Reply:
correct, it's not part of the build that occurs when people import the packaged crate